File: peek_plugin_generic_diagram_menu/_private/tuples/AddIntValueActionTuple.py
from vortex.Tuple import addTupleType, TupleField
from vortex.TupleAction import TupleActionABC

from peek_plugin_generic_diagram_menu._private.PluginNames import genericDiagramMenuTuplePrefix


@addTupleType
class AddIntValueActionTuple(TupleActionABC):
    __tupleType__ = genericDiagramMenuTuplePrefix + "AddIntValueActionTuple"

    genericDiagramMenuId = TupleField()
    offset = TupleField()

File: peek_plugin_generic_diagram_menu/plugin-module/_private/tuples/AddIntValueActionTuple.ts
 import {addTupleType, Tuple, TupleActionABC} from "@synerty/vortexjs";
import {genericDiagramMenuTuplePrefix} from "../PluginNames";

@addTupleType
export class AddIntValueActionTuple extends TupleActionABC {
    static readonly tupleName = genericDiagramMenuTuplePrefix + "AddIntValueActionTuple";

    genericDiagramMenuId: number;
    offset: number;

    constructor() {
        super(AddIntValueActionTuple.tupleName)
    }
}